jenkins/sparklingWaterPipeline.groovy

```diff
@@ -10,21 +10,18 @@ def getS3Path(config) {
 
 String getNightlyVersion(config) {
     def sparkMajorVersion = config.sparkMajorVersion
-    def versionLine = readFile("gradle.properties").split("\n").find() { line -> line.startsWith('version') }
-    def version = versionLine.split("=")[1]
-    if (config.uploadNightly.toBoolean()) {
```

Comment:
no need to have this condition, this method is called only from uploadNightly stage